domain/src/test/java/camp/nextstep/edu/calculator/domain/ExpressionItemsTest.kt

```diff
@@ -0,0 +1,77 @@
+package camp.nextstep.edu.calculator.domain
+
+import camp.nextstep.edu.calculator.domain.ExpressionItem.Operand
+import camp.nextstep.edu.calculator.domain.ExpressionItem.Operation
+import com.google.common.truth.Truth.assertThat
+import org.junit.Before
+import org.junit.Test
+
+class ExpressionItemsTest {
+    private lateinit var expressionItems: ExpressionItems
+
+    @Before
+    fun initialize() {
+        expressionItems = ExpressionItems(emptyList())
+    }
+
+    @Test
+    fun `피연산자 1을추가하고 반환값이 1인지 확인`() {
+        val actual = (expressionItems + Operand(1)).getText()
+        println("actual: $actual")
+
+        assertThat(actual).isEqualTo("1")
+    }
+
+    @Test
+    fun `연산자 + 을추가하고 반환값이 빈문자인지 확인`() {
+        val actual = (expressionItems + Operation.Addition).getText()
+
+        assertThat(actual).isEqualTo("")
+    }
+
+    @Test
+    fun `피연산자 1을추가하고 연산자 + 을추가하고 반환문자가 1 + 인지 확인`() {
+        val actual = (expressionItems + Operand(1) + Operation.Addition).getText()
+
+        assertThat(actual).isEqualTo("1 +")
+    }
+
+    @Test
+    fun `피연산자 1을추가하고 연산자 + 을추가하고 피연산자 2를추가하고 반환값이 1 + 2 인지 확인`() {
+        val actual = (
+            expressionItems +
+                Operand(1) +
+                Operation.Addition +
+                Operand(2)
+            ).getText()
+
+        assertThat(actual).isEqualTo("1 + 2")
+    }
+
+    @Test
+    fun `피연산자 1을추가하고 연산자 +을추가하고 피연산자 2를추가하고 삭제를 호출하여 반환값이 1 + 인지 확인`() {
+        expressionItems = expressionItems + Operand(1) + Operation.Addition + Operand(2)
+
+        val actual = expressionItems.removeLastExpression().getText()
+
+        assertThat(actual).isEqualTo("1 +")
+    }
```

Comment:
개인적으로는 테스트 케이스에서 given, when, then 절을 활용하는 편입니다!  
준비, 실행, 검증 부를 나눠서 작성하면, 무엇을 테스트하는지 조금도 명확해지는거 같단 생각이 들어요!

```suggestion
    fun `1 + 2의 표현식이 주어졌을때, 삭제하면, 반환값이 1 + 이여야한다`() {
        // given : 1 + 2
        expressionItems = ExpressionItems(listOf(Operand(1), Operation.Addition, Operand(2)))

        // when : removeLastExpression
        val actual = expressionItems.removeLastExpression()
        
        // then : 1 +
        assertThat(actual.getText()).isEqualTo("1 +")
    }
```